cypress/integration/projectSettings/project_settings.ts

```diff
@@ -946,6 +952,28 @@ describe("Attaching Spruce to a repo", () => {
   });
 });
 
+describe("New Project button for project", () => {
+  beforeEach(() => {
+    cy.login();
+    cy.visit(getGeneralRoute(project));
+    cy.dataCy("new-project-menu").should("not.exist");
+    cy.dataCy("new-project-button").click();
+    cy.dataCy("new-project-menu").should("be.visible");
+  });
+
+  it("clicking the 'Create New Project' menu button opens the create project modal and closes the menu", () => {
+    cy.dataCy("create-project-button").click();
+    cy.dataCy("new-project-menu").should("not.exist");
+    cy.dataCy("create-project-modal").should("be.visible");
+  });
+
+  it("clicking the 'Duplicate Project' menu button opens the create project modal and closes the menu", () => {
```

Comment:
Wonder if we should also delete these once the flaky tests are fixed? Imo if something can be tested thoroughly in Jest then I would be okay, or maybe even prefer, to leave it out of Cypress . If so, maybe we can add a note in the flaky tests ticket!